rskj-core/src/main/resources/expected.conf

```diff
@@ -235,6 +235,8 @@ rpc = {
             }
         }
     ]
+
```

Comment:
would remove this space here to preserve the style.

Reply:
done